source/reference/methods.txt

```diff
@@ -709,6 +714,24 @@ Native Methods
 
      - Description
 
+   * - ``print()``
+
+     - Print the specified text or variable. ``print()`` and ``printjson()``
+       are aliases for ``console.log()``.
+
+       .. code-block:: code
+
+          > print("hello world")
+          hello world
+
+          > x = "example text"
+          > print(x)
+          example text
+
+   * - :method:`sleep()`
```

Comment:
This links me to the top of the current page. I assume this isn't intentional?

Reply:
Fixed.